src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,40 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+    final int TWELVE = 12;
+    final int MINUS_ONE = -1;
+    final int ELEVEN = 11;
+    private final int[] numbers = new int[TWELVE];
+
+
+    private int total = MINUS_ONE;
+
+    public int getTotal() {return total;}
+
+    public void countIn(int in) {
+        if (!isFull())
+            numbers[++total] = in;
+    }
+
+    public boolean callCheck() {
+        return total == MINUS_ONE;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).
This constant should be different (separate) than the one in peekaboo and countOut methods.